test/test.js

```diff
@@ -129,6 +129,33 @@ describe('simplerdf', () => {
     assert(simple.isArray(posts))
   })
 
+  it('getter should use Arrays if there are multiple values', () => {
+    let graph = blogGraph.clone()
+
+    graph.add(rdf.createTriple(
+      rdf.createNamedNode(blogIri),
+      rdf.createNamedNode('http://schema.org/name'),
+      rdf.createLiteral('simple blog second name')))
+
+    let blog = simple(blogContext, blogIri, graph)
+
+    let names = blog.name
+
+    assert(simple.isArray(names))
```

Comment:
Yes, but that test works as intended. We will always have a drawback, if the predicate is not defined as an array, but there are two or more triples with that predicate in an existing graph.

I see two solutions:

1. Treat that predicate like there is only one triple, which could lead into data loss.
2. Return a unexpected data type, which must be handled in the code.

So what's the smaller problem? I vote for 2. and also JSON-LD does it that way.

The best solution would be validating the data when it's generated. But sometimes you can't control that, but than you should be prepared for "strange" data.